tests/caasp/stack_controller.pm

```diff
@@ -80,13 +80,20 @@ sub velum_bootstrap {
     # Wait until warning messages disappears
     wait_still_screen;
 
-    # Click next button to 'Confirm bootstrap' page
-    send_key_until_needlematch 'velum-next', 'pgdn', 2, 5;
-    assert_and_click 'velum-next';
-
-    # Click bootstrap button
-    assert_screen 'velum-confirm-bootstrap';
-    assert_and_click "velum-bootstrap";
+    if (check_var('DISTRI', 'caasp') && !check_var('VERSION', '1.0')) {
```

Comment:
This version checking approach won't work for long. Better implement a function "caasp_version_at_least" as e.g. "sle_version_at_least" in lib/utils.pm . not a problem now but later so won't block this PR

Reply:
Agreed, as soon as all the three versions are online (Kubic, QAM CaaSP, QA CaaSP) we have to find a better way to keep them in sync ;)